tests/test_http_client.py

```diff
@@ -11,10 +11,10 @@
 
 class StripeClientTestCase(object):
     REQUEST_LIBRARIES = [
-        ("urlfetch", "stripe.http_client.urlfetch"),
-        ("requests", "stripe.http_client.requests"),
-        ("pycurl", "stripe.http_client.pycurl"),
-        ("urllib.request", "stripe.http_client.urllibrequest"),
+        ("urlfetch", "stripe._http_client.urlfetch"),
```

Comment:
As this and util tests show the change might be breaking if users were patching the sdk internals.